src/repository/entity/CatEntity.java

```diff
@@ -0,0 +1,59 @@
+package repository.entity;
+
+import java.util.Objects;
+
+public class CatEntity extends BaseEntity {
+    private String Breed;
+
+    private String Color;
+
+    public CatEntity(String Breed, String Color) {
+
+        this.Breed = Breed;
+        this.Color = Color;
+    }
+
+    public String getBreed() {
+        return Breed;
+    }
+
+    public void setBreed(String Breed) {
+        this.Breed = Breed;
+    }
+
+    public String getColor() {
+        return Color;
+    }
+
+    public void setColor(String Color) {
+        this.Color = Color;
+    }
+
+
+    @Override
+    public String toString() {
+        return "Breed: " + this.Breed + ", Color: " + this.Color;
```

Comment:
It is a good practice to add class name in toString method
So toString method would be like, "ClassName(breed = some, color = some)"